jujugui/static/gui/src/test/test_app.js

```diff
@@ -650,6 +650,7 @@ describe('File drag over notification system', function() {
     });
 
     it('avoids trying to login without credentials', function(done) {
+      sessionStorage.clear();
```

Comment:
Phantom appears to store sessionStorage across sessions. So that's why the first test run passed but the others then failed. This makes sure it clears the sessionstorage before the test which requires there to be no credentials.